map/routing_manager.cpp

```diff
@@ -1002,8 +1002,10 @@ void RoutingManager::BuildRoute(uint32_t timeoutSec)
   // Show preview.
   m2::RectD rect = ShowPreviewSegments(routePoints);
   rect.Scale(kRouteScaleMultiplier);
-  m_drapeEngine.SafeCall(&df::DrapeEngine::SetModelViewRect, rect, true /* applyRotation */,
-                         -1 /* zoom */, true /* isAnim */, true /* useVisibleViewport */);
+
+  //XXX: Uncomment next two lines to enable back zoom on route point add/remove
+  //m_drapeEngine.SafeCall(&df::DrapeEngine::SetModelViewRect, rect, true /* applyRotation */,
+  //                       -1 /* zoom */, true /* isAnim */, true /* useVisibleViewport */);
```

Comment:
- also comment rect calculation, it's not needed
- remove "XXX:", better put the link on discussion (if it has some issue), or your initial PR with distances to explain the problem

Reply:
Commented out rect calculation. Changed comments. Added link to issue #5409.